src/doc/sprite.h

```diff
@@ -272,6 +275,9 @@ class Sprite : public WithUserData {
   // giving the possibility to handle tiles exclusively to a plugin.
   std::string m_tileManagementPlugin;
 
+  // This setting indicates if the layers of this sprite are using UUIDs.
+  bool m_useUuidsForLayers = false;
```

Comment:
Probably this and getter/setters could be called just `uuidsForLayers` (removing the "use" verb).